src/blueprints/tv2/enums/tv2-piece-type.ts

```diff
@@ -0,0 +1,16 @@
+export enum Tv2PieceType {
+  CAMERA = 'CAMERA',
+  REPLAY = 'REPLAY',
+  REMOTE = 'REMOTE',
+  GRAPHICS = 'GRAPHICS',
+  OVERLAY_GRAPHICS = 'OVERLAY_GRAPHICS',
```

Comment:
Should e.g. an `Ident` use this one, or create a new one that is even more specific? i.e a `IDENT_OVERLAY_GRAPHICS`, or something along those lines.

Reply:
I would say that it should use `OVERLAY_GRAPHICS`, and then if it needs further specification, then one could add a new metadata attribute `overlayGraphicsType`.